src/lib/promptForUserId.ts

```diff
@@ -5,5 +5,14 @@ export async function promptForUserId(): Promise<string> {
   // Ignore failure since we're about to prompt for it
   const userId = await getUserId();
 
-  return getPrompt('userid', userId, 'Network Username', null);
+  return getPrompt(
+    'userid',
+    userId,
+    'Active Directory Username (this is not part of your email)',
+    null
+  );
```

Comment:
It's not always clear to users where they can find this. Maybe in the documentation we can say something about finding this information on the ALKS users tab.